snap/snapcraft.yaml

```diff
@@ -26,48 +26,130 @@ confinement: strict
     plugs: [network, network-bind]
-  vault:
-    command: bin/vault
+  core-config-seed:
+    command: bin/fork-wrapper.sh $SNAP/bin/go-services-wrapper.sh config-seed -c ${SNAP_DATA}/config
     plugs: [network, network-bind]
-  cassandra:
-    command: bin/cassandra-wrapper.sh
     daemon: forking
-    # mount-observe needs to be manually connected post-install:
-    # `sudo snap connect cassandra:mount-observe ubuntu-core:mount-observe`
-    plugs:
+    passthrough:
+      after:
+        - consul
+  mongod:
+    command: bin/mongod-wrapper.sh
+    stop-command: bin/mongod --shutdown --dbpath ${SNAP_DATA}/mongo/db
+    plugs: [hardware-observe, network, network-bind, system-observe]
+    daemon: simple
+    passthrough:
+      after:
+        - core-config-seed
+  mongo-worker:
+    command: bin/mongo-worker-wrapper.sh
+    plugs: [network]
+    daemon: oneshot
+    passthrough:
+      after:
+        - mongod
+  security-services:
+    command: bin/fork-wrapper.sh $SNAP/bin/security-start.sh
+    stop-command: bin/security-stop.sh
+    daemon: forking
+    plugs: 
       - network
       - network-bind
-      # Needed for checking available space in the data directories
-      # (java.io.File.getTotalSpace).
       - mount-observe
-  core-config-seed:
-    command: bin/start-config-seed.sh
+    passthrough:
+      after:
+        - core-config-seed
+  support-logging:
+    command: bin/fork-wrapper.sh $SNAP/bin/go-services-wrapper.sh support-logging --consul
+    daemon: forking
     plugs: [network, network-bind]
-  start-edgex:
-    command: bin/start-edgex.sh
-    plugs: [network, network-bind, system-observe, process-control, hardware-observe]
-  stop-edgex:
-    command: bin/stop-edgex.sh
-    plugs: [system-observe, process-control]
-  security-start:
-    command: bin/security-start.sh
-    plugs:
-      - network
-      - network-bind
+    passthrough:
+      after:
+        - core-config-seed
+        - mongo-worker
+  support-notifications:
+    command: bin/fork-wrapper.sh $SNAP/bin/go-services-wrapper.sh support-notifications --consul
+    daemon: forking
+    plugs: [network, network-bind]
+    passthrough:
+      after:
+        - core-config-seed
+        - mongo-worker
+  core-data:
+    command: bin/fork-wrapper.sh $SNAP/bin/go-services-wrapper.sh core-data --consul
+    daemon: forking
+    plugs: [network, network-bind]
+    passthrough:
+      after:
+        - core-config-seed
+        - mongo-worker
+  core-metadata:
+    command: bin/fork-wrapper.sh $SNAP/bin/go-services-wrapper.sh core-metadata --consul
+    daemon: forking
+    plugs: [network, network-bind]
+    passthrough:
+      after:
+        - core-config-seed
+        - mongo-worker
+  core-command:
+    command: bin/fork-wrapper.sh $SNAP/bin/go-services-wrapper.sh core-command --consul
+    daemon: forking
+    plugs: [network, network-bind, ]
+    passthrough:
+      after:
+        - core-config-seed
+        - mongo-worker
+  support-rulesengine:
```

Comment:
Remove...

Reply:
yes my cherry-pick must have picked this back up as well